Batteries/Data/List/Lemmas.lean

```diff
@@ -478,6 +478,20 @@ theorem length_eraseIdx : ∀ {l i}, i < length l → length (@eraseIdx α l i)
 
 @[simp] theorem length_tail (l : List α) : length (tail l) = length l - 1 := by cases l <;> rfl
 
+theorem succ_length_tail (l : List α) (h : 0 < length l) : (length (tail l)).succ = length l := by
+  cases l <;> simp at h ⊢
+
+theorem length_tail_le (l : List α) : length (tail l) ≤ length l := by
+  cases l <;> simp [Nat.le_succ]
+
+@[simp] theorem getElem?_tail {n} (l : List α) : l.tail[n]? = l[n + 1]? := by cases l <;> simp
```

Comment:
Batteries uses auto-implicits so you should either have `(n)` or leave it out entirely.